### DISCUSSION TITLE
Adding fields to an already existing tiddler using tiddlyclip

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#72] [Q&A] [answered]
I want to know whether its possible to add fields to an already existing tiddler ?

I was using [this plug in to add cover page to tiddlers like in notion](https://tw-cpl.netlify.app/#Gk0Wk%2Fnotionpage-covericon:Index%20Gk0Wk%2Fnotionpage-covericon). I was planning to use tiddlyclip to add a field called `page-cover` with the field value being extracted from the `@imageURL`

Is it possible to add field to an already existing tiddler using tiddlyclip

**Comment by buggyj** [comment, marked as answer]
Yes you can add fields to existing tiddlers

This needs 2 steps:

You would need the title of the tiddler to modify to be loaded into tiddlyclip, before clipping.

There is the command to copy text from a webpage into a variable in the rule below

First hightlight the tiddler name and use tiddlyclip to load into the variable '@cptext'

![image](https://user-images.githubusercontent.com/2855795/152678472-237d60af-e89b-414d-be75-8257cff8a57a.png)

This will then be used in the 'AddField' rule.

Add this to your tiddlyclipconfig

|CopyToBrowser||||cptext|
|AddField|||[[AddField]]||

and import this tiddler

[AddField.json.zip](https://github.com/buggyj/tiddlyclip-plugin/files/8010034/AddField.json.zip)

**Reply by arunnbabu81** [reply]
> Yes you can add fields to existing tiddlers
> 
> This needs 2 steps:
> 
> You would need the title of the tiddler to modify to be loaded into tiddlyclip, before clipping.
> 
> There is the command to copy text from a webpage into a variable in the rule below
> 
> First hightlight the tiddler name and use tiddlyclip to load into the variable '@cptext'
> 
> ![image](https://user-images.githubusercontent.com/2855795/152678472-237d60af-e89b-414d-be75-8257cff8a57a.png)
> 
> This will then be used in the 'AddField' rule.
> 
> Add this to your tiddlyclipconfig
> 
> |CopyToBrowser||||cptext| |AddField|||[[AddField]]||
> 
> and import this tiddler
> 
> [AddField.json.zip](https://github.com/buggyj/tiddlyclip-plugin/files/8010034/AddField.json.zip)

This one will do the trick for me. Thank you
